### ISSUE TITLE
sstabledump: add support for encrypted SSTables

### ISSUE THREAD

**Issue by vladzcloudius** [scylladb/scylla-tools-java#265]
HEAD: fdc8fcc22cf94cf35d51776ca4a76b924cd0e42f

*Description*
We need to be able to dump SSTables with encryption enabled. Today there is no such ability and sstabledump is a very powerful tool that is very useful in the field (e.g. when one needs to debug tombstones related issues).

**Comment by vladzcloudius** [comment]
@raphaelsc @slivne @eliransin FYI

**Comment by fruch** [comment]
@denesb I'm guessing we can close this one, in favor of `scylla sstable` ?

**Comment by denesb** [comment]
> @denesb I'm guessing we can close this one, in favor of `scylla sstable` ?

Yes, see https://github.com/scylladb/scylla-enterprise/issues/3210.
